web/src/components/map.js

```diff
@@ -325,7 +325,9 @@ export default class Map {
 
   setCo2color(arg, theme) {
     this.co2color = arg;
-    this.theme = theme;
+    if (theme) {
+      this.theme = theme;
+    }
```

Comment:
does the map still work if this.theme is left undefined?

Reply:
The theme is actually _always_ set in the constructor (see below); this PR only makes the overrides in `setCo2color` conditional.

https://github.com/tmrowco/electricitymap-contrib/blob/77b70262ec4b2a01a5d860d13672eb400947f59c/web/src/main.js#L308